aspnetcore/fundamentals/minimal-apis/responses.md

```diff
@@ -81,6 +81,39 @@ app.MapGet("/hello", () => TypedResults.Ok(new Message() {  Text = "Hello World!
 
 For more information about describing a response type, see [OpenAPI support in minimal APIs](/aspnet/core/fundamentals/minimal-apis/openapi#describe-response-types-1).
 
+As mentioned previously, when using `TypedResults`, a conversion is not needed. Consider the following minimal API which returns a `TypedResults` class
+
+<!-- TODO, https://github.com/dotnet/AspNetCore.Docs/issues/28916 moves the following code to a project -->
+
+```csharp
+public static async Task<IResult> GetAllTodos(TodoDb db)
```

Comment:
```suggestion
public static async Task<Ok<Todo[]>> GetAllTodos(TodoDb db)
```

Note that this example works the same with `Results.Ok()` as even though the method returns `IResult`, the concrete type can still be checked the same way in the test.